banman/reason.go

```diff
@@ -0,0 +1,43 @@
+package banman
+
+// Reason includes the different possible reasons which caused us to ban a peer.
+type Reason uint8
+
+// We prevent using `iota` to ensure the order does not have the value since
+// these are serialized within the database.
+const (
```

Comment:
Seems like some peers are `Unresponsive` which could be another `Reason`.

Reply:
See other comment.